Assets/Scripts/Warrior.cs

```diff
@@ -17,14 +15,14 @@ public class Warrior : MonoBehaviour
     [SerializeField] private float _rotationSpeed;
 
     private GameManager _gameManager;
+    private Warrior _currentEnemy;
     private Animator _warriorAnimator;
     private Quaternion _bulletQuaternion;
 
     private float _targetRotation;
     private float _startRotation;
     private float _shootingTimer;
     private float _deathExplosionTimer = 5f;
```

Comment:
use for initialization TimeForDeath const

Reply:
done